omics/omics_uri_agent/uri_parser.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+from urllib.parse import urlparse
+from omics.transfer import (
+    ReadSetFileName,
+    ReferenceFileName
+)
+
+
+# Example URIs supported:
```

Comment:
Let's put documentation like this into the README along with a description of how to use the new tool.